src/modules/70-pipeline/v1/components/RunPipelineModalV1/CICodebaseInputSetFormV1.tsx

```diff
@@ -56,19 +56,22 @@ export enum ConnectionType {
 export const buildTypeInputNames: Record<string, string> = {
```

Comment:
Could you check the uses of this const? This variable is not anywhere in the present develop
```suggestion
export const buildTypeInputNames: Record< BuildCodebaseType, string> = {
```

Reply:
will refactor in next pr